myParser.py

```diff
@@ -40,6 +40,12 @@ def parse():
         required=True,
         help="Date of departure flight.")
     
+    parser.add_argument(
+        "--flightnum",
+        "-ft",
```

Comment:
Flightnum should probably be abbreviated as fn.

Reply:
Ok